### PR TITLE
#165305752 Update swagger description messages

### PR DESCRIPTION
**Pull request by Amoswachira**
#### What does this PR do?
Adds a description to the swagger endpoints that don't have any description
#### Description of Task to be completed?
make sure the following endpoints in the swagger documentation have an updated description
```POST /api/users/```
```POST /api/users/login/ ```
### Local Development Setup
 - First Create python virtual env
 ```
  $ virtualenv -p python .venv
 ```
 - Install Requirements
 ```
  $ pip install -r requiremts.txt
 ```
 - Copy .env-example to .env and set config
 ```
  $ cp .env-example .env
 ```
 - Create postgres database
 ```
  $ sudo su postgres
  $ psql
  postgres=# CREATE USER django WITH PASSWORD 'password';
  postgres=# ALTER ROLE django SET client_encoding TO 'utf8';
  postgres=# ALTER ROLE django SET default_transaction_isolation TO 'read committed';
  postgres=# ALTER ROLE django SET timezone TO 'UTC';
  postgres=# CREATE DATABASE drf;
  postgres=# GRANT ALL PRIVILEGES ON DATABASE drf TO django;
  postgres=# \q
  $ exit
````
**After cloning the repo:-**
```
$ git checkout  ft-update-swagger-description-165305752

 ```
- Run Server

  $ python manage.py runserver

-Then open the following endpoint on the browser

  http://127.0.0.1:8000/api/swagger/#/

#### What are the relevant pivotal tracker stories?

 [#165305752](https://www.pivotaltracker.com/n/projects/2326460/stories/165305752)

#### Screenshots (if appropriate)
Screenshot 2019-05-07 at 17 53 03

### CODE REVIEW

File: authors/apps/authentication/views.py [hunk 1]
```diff
@@ -75,6 +81,9 @@ def post(self, request):
 
 
 class UserRetrieveUpdateAPIView(RetrieveUpdateAPIView):
+    """
```
Comment (oluwagbenga-joloko) [inline authors/apps/authentication/views.py:84]: can you add the docstrings to each method in the class as well, refer to the implementation used in the profiles app as a guide.  @Amoswachira

Comment (Amoswachira) [reply authors/apps/authentication/views.py:84]: @oluwagbenga-joloko okay let me update that

Comment (Amoswachira) [reply authors/apps/authentication/views.py:84]: @oluwagbenga-joloko I have added the docstrings

**Review by Ogutu-Brian — APPROVED** [review]
LGTM

**Review by IssaIan — APPROVED** [review]

**Pull request merged**